mysite/checkers/game.py

```diff
@@ -38,9 +38,10 @@ def __init__(self):
 		self.selected_piece = None # a board location. 
 		self.hop = False
 		self.selected_legal_moves = []
+		self.winner  = ''
 
```

Comment:
if the function has no purpose we should not have it in our master branch code base

Reply:
removed that funtion.!